[2022-01-14 20:21:37,784] {processor.py:163} INFO - Started process (PID=20592) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:21:37,817] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:21:37,818] {logging_mixin.py:109} INFO - [2022-01-14 20:21:37,818] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:21:38,437] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:21:41,577] {logging_mixin.py:109} INFO - [2022-01-14 20:21:41,575] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:21:41,879] {logging_mixin.py:109} INFO - [2022-01-14 20:21:41,876] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-13 00:00:00+00:00
[2022-01-14 20:21:41,978] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 4.450 seconds
[2022-01-14 20:22:13,177] {processor.py:163} INFO - Started process (PID=20938) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:22:13,238] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:22:13,479] {logging_mixin.py:109} INFO - [2022-01-14 20:22:13,479] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:22:14,270] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:22:14,432] {logging_mixin.py:109} INFO - [2022-01-14 20:22:14,432] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:22:15,287] {logging_mixin.py:109} INFO - [2022-01-14 20:22:15,287] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-13 00:00:00+00:00
[2022-01-14 20:22:15,723] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 2.590 seconds
[2022-01-14 20:22:45,891] {processor.py:163} INFO - Started process (PID=21132) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:22:45,905] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:22:45,911] {logging_mixin.py:109} INFO - [2022-01-14 20:22:45,911] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:22:46,054] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:22:46,094] {logging_mixin.py:109} INFO - [2022-01-14 20:22:46,093] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:22:46,167] {logging_mixin.py:109} INFO - [2022-01-14 20:22:46,167] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-13 00:00:00+00:00
[2022-01-14 20:22:46,211] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.322 seconds
[2022-01-14 20:23:16,358] {processor.py:163} INFO - Started process (PID=21241) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:23:16,359] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:23:16,360] {logging_mixin.py:109} INFO - [2022-01-14 20:23:16,360] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:23:16,387] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:23:16,393] {logging_mixin.py:109} INFO - [2022-01-14 20:23:16,393] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:23:16,407] {logging_mixin.py:109} INFO - [2022-01-14 20:23:16,407] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-13 00:00:00+00:00
[2022-01-14 20:23:16,414] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.061 seconds
[2022-01-14 20:23:46,683] {processor.py:163} INFO - Started process (PID=21291) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:23:46,684] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:23:46,685] {logging_mixin.py:109} INFO - [2022-01-14 20:23:46,685] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:23:46,887] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:23:46,976] {logging_mixin.py:109} INFO - [2022-01-14 20:23:46,976] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:23:47,132] {logging_mixin.py:109} INFO - [2022-01-14 20:23:47,132] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-13 00:00:00+00:00
[2022-01-14 20:23:47,229] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.552 seconds
[2022-01-14 20:24:17,774] {processor.py:163} INFO - Started process (PID=21360) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:24:17,779] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:24:17,783] {logging_mixin.py:109} INFO - [2022-01-14 20:24:17,783] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:24:18,036] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:24:18,085] {logging_mixin.py:109} INFO - [2022-01-14 20:24:18,085] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:24:18,183] {logging_mixin.py:109} INFO - [2022-01-14 20:24:18,183] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-13 00:00:00+00:00
[2022-01-14 20:24:18,270] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.535 seconds
[2022-01-14 20:24:48,686] {processor.py:163} INFO - Started process (PID=21553) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:24:48,719] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:24:48,721] {logging_mixin.py:109} INFO - [2022-01-14 20:24:48,720] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:24:48,915] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:24:48,930] {logging_mixin.py:109} INFO - [2022-01-14 20:24:48,929] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:24:49,022] {logging_mixin.py:109} INFO - [2022-01-14 20:24:49,022] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-13 00:00:00+00:00
[2022-01-14 20:24:49,079] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.401 seconds
[2022-01-14 20:33:00,452] {processor.py:163} INFO - Started process (PID=22736) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:33:00,455] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:33:00,456] {logging_mixin.py:109} INFO - [2022-01-14 20:33:00,456] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:33:00,547] {logging_mixin.py:109} INFO - [2022-01-14 20:33:00,546] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:33:00,897] {processor.py:690} INFO - Deactivated 1 DAGs which are no longer present in /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:33:00,899] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:33:01,052] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.741 seconds
[2022-01-14 20:33:32,247] {processor.py:163} INFO - Started process (PID=23036) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:33:32,250] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:33:32,251] {logging_mixin.py:109} INFO - [2022-01-14 20:33:32,251] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:33:32,850] {logging_mixin.py:109} INFO - [2022-01-14 20:33:32,800] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:33:33,110] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:33:33,556] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 1.658 seconds
[2022-01-14 20:34:04,012] {processor.py:163} INFO - Started process (PID=23218) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:34:04,046] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:34:04,046] {logging_mixin.py:109} INFO - [2022-01-14 20:34:04,046] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:34:04,096] {logging_mixin.py:109} INFO - [2022-01-14 20:34:04,095] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:34:04,448] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:34:04,606] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.607 seconds
[2022-01-14 20:34:34,842] {processor.py:163} INFO - Started process (PID=23298) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:34:34,846] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:34:34,847] {logging_mixin.py:109} INFO - [2022-01-14 20:34:34,847] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:34:34,860] {logging_mixin.py:109} INFO - [2022-01-14 20:34:34,859] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:34:34,942] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:34:35,041] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.231 seconds
[2022-01-14 20:35:05,442] {processor.py:163} INFO - Started process (PID=23374) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:35:05,443] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:35:05,444] {logging_mixin.py:109} INFO - [2022-01-14 20:35:05,444] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:35:05,453] {logging_mixin.py:109} INFO - [2022-01-14 20:35:05,453] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:35:05,541] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:35:05,600] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.160 seconds
[2022-01-14 20:35:35,987] {processor.py:163} INFO - Started process (PID=23424) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:35:35,988] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:35:35,989] {logging_mixin.py:109} INFO - [2022-01-14 20:35:35,989] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:35:35,997] {logging_mixin.py:109} INFO - [2022-01-14 20:35:35,997] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:35:36,058] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:35:36,100] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.151 seconds
[2022-01-14 20:36:06,395] {processor.py:163} INFO - Started process (PID=23474) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:36:06,398] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:36:06,400] {logging_mixin.py:109} INFO - [2022-01-14 20:36:06,399] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:36:06,439] {logging_mixin.py:109} INFO - [2022-01-14 20:36:06,439] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:36:06,528] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:36:06,549] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.159 seconds
[2022-01-14 20:36:36,700] {processor.py:163} INFO - Started process (PID=23516) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:36:36,703] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:36:36,705] {logging_mixin.py:109} INFO - [2022-01-14 20:36:36,705] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:36:36,856] {logging_mixin.py:109} INFO - [2022-01-14 20:36:36,852] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:36:37,033] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:36:37,096] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.401 seconds
[2022-01-14 20:37:07,246] {processor.py:163} INFO - Started process (PID=23566) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:37:07,248] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:37:07,249] {logging_mixin.py:109} INFO - [2022-01-14 20:37:07,249] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:37:07,289] {logging_mixin.py:109} INFO - [2022-01-14 20:37:07,288] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:37:07,334] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:37:07,345] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.104 seconds
[2022-01-14 20:37:37,992] {processor.py:163} INFO - Started process (PID=23690) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:37:37,997] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:37:37,998] {logging_mixin.py:109} INFO - [2022-01-14 20:37:37,998] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:37:38,187] {logging_mixin.py:109} INFO - [2022-01-14 20:37:38,186] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:37:40,382] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:37:40,499] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 2.517 seconds
[2022-01-14 20:38:11,340] {processor.py:163} INFO - Started process (PID=23990) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:38:11,342] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:38:11,344] {logging_mixin.py:109} INFO - [2022-01-14 20:38:11,343] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:38:11,489] {logging_mixin.py:109} INFO - [2022-01-14 20:38:11,488] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:38:11,784] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:38:11,835] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.504 seconds
[2022-01-14 20:38:42,699] {processor.py:163} INFO - Started process (PID=24056) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:38:42,789] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:38:42,791] {logging_mixin.py:109} INFO - [2022-01-14 20:38:42,790] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:38:42,897] {logging_mixin.py:109} INFO - [2022-01-14 20:38:42,895] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:38:44,683] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:38:45,133] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 2.440 seconds
[2022-01-14 20:39:16,196] {processor.py:163} INFO - Started process (PID=24180) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:39:16,199] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:39:16,199] {logging_mixin.py:109} INFO - [2022-01-14 20:39:16,199] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:39:16,385] {logging_mixin.py:109} INFO - [2022-01-14 20:39:16,384] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:39:16,642] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:39:16,698] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.509 seconds
[2022-01-14 20:39:47,496] {processor.py:163} INFO - Started process (PID=24364) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:39:47,580] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:39:47,583] {logging_mixin.py:109} INFO - [2022-01-14 20:39:47,583] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:39:47,597] {logging_mixin.py:109} INFO - [2022-01-14 20:39:47,597] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:39:47,990] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:39:48,086] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.594 seconds
[2022-01-14 20:40:18,730] {processor.py:163} INFO - Started process (PID=25857) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:40:18,737] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:40:18,737] {logging_mixin.py:109} INFO - [2022-01-14 20:40:18,737] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:40:18,751] {logging_mixin.py:109} INFO - [2022-01-14 20:40:18,751] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:40:18,784] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:40:18,799] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.080 seconds
[2022-01-14 20:40:19,797] {processor.py:163} INFO - Started process (PID=25911) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:40:19,798] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:40:19,798] {logging_mixin.py:109} INFO - [2022-01-14 20:40:19,798] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:40:19,819] {logging_mixin.py:109} INFO - [2022-01-14 20:40:19,818] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:40:19,840] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:40:19,851] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.056 seconds
[2022-01-14 20:40:52,235] {processor.py:163} INFO - Started process (PID=26515) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:40:52,243] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:40:52,246] {logging_mixin.py:109} INFO - [2022-01-14 20:40:52,246] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:40:53,833] {logging_mixin.py:109} INFO - [2022-01-14 20:40:53,692] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:40:55,631] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:40:56,334] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 4.194 seconds
[2022-01-14 20:44:31,255] {processor.py:163} INFO - Started process (PID=35517) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:44:31,333] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:44:31,334] {logging_mixin.py:109} INFO - [2022-01-14 20:44:31,334] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:44:31,452] {logging_mixin.py:109} INFO - [2022-01-14 20:44:31,452] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:44:33,186] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:44:33,839] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 2.792 seconds
[2022-01-14 20:45:04,446] {processor.py:163} INFO - Started process (PID=35870) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:45:04,447] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:45:04,447] {logging_mixin.py:109} INFO - [2022-01-14 20:45:04,447] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:45:04,555] {logging_mixin.py:109} INFO - [2022-01-14 20:45:04,554] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:45:06,760] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:45:07,301] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 2.942 seconds
[2022-01-14 20:45:38,108] {processor.py:163} INFO - Started process (PID=36409) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:45:38,154] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:45:38,446] {logging_mixin.py:109} INFO - [2022-01-14 20:45:38,160] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:45:38,742] {logging_mixin.py:109} INFO - [2022-01-14 20:45:38,708] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:45:39,456] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:45:40,306] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 2.204 seconds
[2022-01-14 20:46:10,848] {processor.py:163} INFO - Started process (PID=38379) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:46:10,851] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:46:10,856] {logging_mixin.py:109} INFO - [2022-01-14 20:46:10,856] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:46:11,059] {logging_mixin.py:109} INFO - [2022-01-14 20:46:11,059] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:46:12,402] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:46:12,653] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 1.903 seconds
[2022-01-14 20:46:43,608] {processor.py:163} INFO - Started process (PID=39917) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:46:43,610] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:46:43,610] {logging_mixin.py:109} INFO - [2022-01-14 20:46:43,610] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:46:43,902] {logging_mixin.py:109} INFO - [2022-01-14 20:46:43,899] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:46:45,101] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:46:45,958] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 2.361 seconds
[2022-01-14 20:47:16,707] {processor.py:163} INFO - Started process (PID=43341) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:47:16,710] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:47:16,711] {logging_mixin.py:109} INFO - [2022-01-14 20:47:16,711] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:47:16,764] {logging_mixin.py:109} INFO - [2022-01-14 20:47:16,763] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:47:17,100] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:47:17,141] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.440 seconds
[2022-01-14 20:47:47,356] {processor.py:163} INFO - Started process (PID=46164) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:47:47,359] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:47:47,359] {logging_mixin.py:109} INFO - [2022-01-14 20:47:47,359] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:47:47,644] {logging_mixin.py:109} INFO - [2022-01-14 20:47:47,643] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdag.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdag'
[2022-01-14 20:47:48,009] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:47:48,349] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 1.002 seconds
[2022-01-14 20:48:17,505] {processor.py:163} INFO - Started process (PID=46201) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:48:17,592] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:48:17,597] {logging_mixin.py:109} INFO - [2022-01-14 20:48:17,596] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:48:17,899] {logging_mixin.py:109} INFO - [2022-01-14 20:48:17,898] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdags.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdags.subdag_parallel_dag'
[2022-01-14 20:48:18,243] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:48:18,403] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 0.905 seconds
[2022-01-14 20:48:49,048] {processor.py:163} INFO - Started process (PID=46399) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:48:49,050] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:48:49,051] {logging_mixin.py:109} INFO - [2022-01-14 20:48:49,051] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:48:49,197] {logging_mixin.py:109} INFO - [2022-01-14 20:48:49,195] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdags.subdag_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdags.subdag_parallel_dag'
[2022-01-14 20:48:50,345] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:48:50,491] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 1.448 seconds
[2022-01-14 20:49:00,442] {processor.py:163} INFO - Started process (PID=46487) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:49:00,443] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:49:00,444] {logging_mixin.py:109} INFO - [2022-01-14 20:49:00,444] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:49:00,647] {logging_mixin.py:109} INFO - [2022-01-14 20:49:00,647] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdags.subdags_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdags.subdags_parallel_dag'
[2022-01-14 20:49:01,445] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:49:01,546] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 1.344 seconds
[2022-01-14 20:49:32,097] {processor.py:163} INFO - Started process (PID=46759) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:49:32,101] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:49:32,103] {logging_mixin.py:109} INFO - [2022-01-14 20:49:32,103] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:49:32,294] {logging_mixin.py:109} INFO - [2022-01-14 20:49:32,293] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/parallel_dags.py
Traceback (most recent call last):
  File "/home/airflow/sandbox/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/parallel_dags.py", line 6, in <module>
    from subdags.subdags_parallel_dag import subdag_parallel_dag
ModuleNotFoundError: No module named 'subdags.subdags_parallel_dag'
[2022-01-14 20:49:33,097] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:49:33,242] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 1.148 seconds
[2022-01-14 20:49:54,940] {processor.py:163} INFO - Started process (PID=47017) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:49:54,943] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:49:54,944] {logging_mixin.py:109} INFO - [2022-01-14 20:49:54,944] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:49:55,283] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag.processing_tasks', 'parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:49:57,983] {logging_mixin.py:109} INFO - [2022-01-14 20:49:57,983] {dag.py:2396} INFO - Sync 2 DAGs
[2022-01-14 20:49:58,100] {logging_mixin.py:109} WARNING - /home/airflow/sandbox/lib/python3.8/site-packages/airflow/configuration.py:361 DeprecationWarning: The dag_concurrency option in [core] has been renamed to max_active_tasks_per_dag - the old setting has been used, but please update your config.
[2022-01-14 20:49:58,104] {logging_mixin.py:109} INFO - [2022-01-14 20:49:58,104] {dag.py:2415} INFO - Creating ORM DAG for parallel_dag.processing_tasks
[2022-01-14 20:49:58,303] {logging_mixin.py:109} INFO - [2022-01-14 20:49:58,302] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-13 00:00:00+00:00
[2022-01-14 20:49:58,303] {logging_mixin.py:109} INFO - [2022-01-14 20:49:58,303] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:49:58,588] {logging_mixin.py:109} INFO - [2022-01-14 20:49:58,588] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:49:58,698] {logging_mixin.py:109} INFO - [2022-01-14 20:49:58,698] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:49:58,744] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 3.812 seconds
[2022-01-14 20:50:29,596] {processor.py:163} INFO - Started process (PID=47338) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:50:29,599] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:50:29,599] {logging_mixin.py:109} INFO - [2022-01-14 20:50:29,599] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:50:31,587] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag.processing_tasks', 'parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:50:32,284] {logging_mixin.py:109} INFO - [2022-01-14 20:50:32,283] {dag.py:2396} INFO - Sync 2 DAGs
[2022-01-14 20:50:32,803] {logging_mixin.py:109} INFO - [2022-01-14 20:50:32,803] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-13 00:00:00+00:00
[2022-01-14 20:50:32,988] {logging_mixin.py:109} INFO - [2022-01-14 20:50:32,805] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:50:33,285] {logging_mixin.py:109} INFO - [2022-01-14 20:50:33,285] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:50:33,483] {logging_mixin.py:109} INFO - [2022-01-14 20:50:33,453] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:50:33,901] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 4.399 seconds
[2022-01-14 20:51:04,203] {processor.py:163} INFO - Started process (PID=47663) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:51:04,205] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:51:04,290] {logging_mixin.py:109} INFO - [2022-01-14 20:51:04,290] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:51:04,692] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag.processing_tasks', 'parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:51:04,904] {logging_mixin.py:109} INFO - [2022-01-14 20:51:04,904] {dag.py:2396} INFO - Sync 2 DAGs
[2022-01-14 20:51:05,092] {logging_mixin.py:109} INFO - [2022-01-14 20:51:05,092] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-13 00:00:00+00:00
[2022-01-14 20:51:05,097] {logging_mixin.py:109} INFO - [2022-01-14 20:51:05,097] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:51:05,138] {logging_mixin.py:109} INFO - [2022-01-14 20:51:05,138] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:51:05,153] {logging_mixin.py:109} INFO - [2022-01-14 20:51:05,153] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:51:05,242] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 1.044 seconds
[2022-01-14 20:51:36,405] {processor.py:163} INFO - Started process (PID=48254) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:51:36,792] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:51:36,793] {logging_mixin.py:109} INFO - [2022-01-14 20:51:36,793] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:51:39,439] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag.processing_tasks', 'parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:51:40,643] {logging_mixin.py:109} INFO - [2022-01-14 20:51:40,643] {dag.py:2396} INFO - Sync 2 DAGs
[2022-01-14 20:51:42,293] {logging_mixin.py:109} INFO - [2022-01-14 20:51:42,293] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-14T00:00:00+00:00
[2022-01-14 20:51:42,294] {logging_mixin.py:109} INFO - [2022-01-14 20:51:42,294] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:51:42,541] {logging_mixin.py:109} INFO - [2022-01-14 20:51:42,541] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:51:42,990] {logging_mixin.py:109} INFO - [2022-01-14 20:51:42,990] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:51:43,742] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 7.842 seconds
[2022-01-14 20:52:16,552] {processor.py:163} INFO - Started process (PID=49298) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:52:16,647] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:52:16,651] {logging_mixin.py:109} INFO - [2022-01-14 20:52:16,650] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:52:20,791] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag.processing_tasks', 'parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:52:21,692] {logging_mixin.py:109} INFO - [2022-01-14 20:52:21,692] {dag.py:2396} INFO - Sync 2 DAGs
[2022-01-14 20:52:22,841] {logging_mixin.py:109} INFO - [2022-01-14 20:52:22,840] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-14T00:00:00+00:00
[2022-01-14 20:52:22,841] {logging_mixin.py:109} INFO - [2022-01-14 20:52:22,841] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:52:23,100] {logging_mixin.py:109} INFO - [2022-01-14 20:52:23,099] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:52:23,438] {logging_mixin.py:109} INFO - [2022-01-14 20:52:23,438] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:52:24,189] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 7.836 seconds
[2022-01-14 20:52:55,348] {processor.py:163} INFO - Started process (PID=50544) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:52:55,349] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:52:55,350] {logging_mixin.py:109} INFO - [2022-01-14 20:52:55,350] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:53:00,840] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag.processing_tasks', 'parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:53:03,351] {logging_mixin.py:109} INFO - [2022-01-14 20:53:03,350] {dag.py:2396} INFO - Sync 2 DAGs
[2022-01-14 20:53:05,344] {logging_mixin.py:109} INFO - [2022-01-14 20:53:05,344] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-14T00:00:00+00:00
[2022-01-14 20:53:05,344] {logging_mixin.py:109} INFO - [2022-01-14 20:53:05,344] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:53:06,339] {logging_mixin.py:109} INFO - [2022-01-14 20:53:06,339] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:53:06,941] {logging_mixin.py:109} INFO - [2022-01-14 20:53:06,941] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:53:07,840] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 12.497 seconds
[2022-01-14 20:53:39,990] {processor.py:163} INFO - Started process (PID=51804) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:53:40,889] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:53:40,938] {logging_mixin.py:109} INFO - [2022-01-14 20:53:40,938] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:53:46,342] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag.processing_tasks', 'parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:53:47,691] {logging_mixin.py:109} INFO - [2022-01-14 20:53:47,691] {dag.py:2396} INFO - Sync 2 DAGs
[2022-01-14 20:53:50,238] {logging_mixin.py:109} INFO - [2022-01-14 20:53:50,238] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-14T00:00:00+00:00
[2022-01-14 20:53:50,239] {logging_mixin.py:109} INFO - [2022-01-14 20:53:50,238] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:53:51,286] {logging_mixin.py:109} INFO - [2022-01-14 20:53:50,452] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:53:51,800] {logging_mixin.py:109} INFO - [2022-01-14 20:53:51,800] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:53:52,197] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 12.259 seconds
[2022-01-14 20:54:23,947] {processor.py:163} INFO - Started process (PID=53753) to work on /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:54:24,033] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/parallel_dags.py for tasks to queue
[2022-01-14 20:54:24,039] {logging_mixin.py:109} INFO - [2022-01-14 20:54:24,039] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:54:26,533] {processor.py:654} INFO - DAG(s) dict_keys(['parallel_dag.processing_tasks', 'parallel_dag']) retrieved from /home/airflow/airflow/dags/parallel_dags.py
[2022-01-14 20:54:27,684] {logging_mixin.py:109} INFO - [2022-01-14 20:54:27,684] {dag.py:2396} INFO - Sync 2 DAGs
[2022-01-14 20:54:29,450] {logging_mixin.py:109} INFO - [2022-01-14 20:54:29,450] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag to 2022-01-14T00:00:00+00:00
[2022-01-14 20:54:29,451] {logging_mixin.py:109} INFO - [2022-01-14 20:54:29,451] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:54:29,886] {logging_mixin.py:109} INFO - [2022-01-14 20:54:29,886] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-14 20:54:30,441] {logging_mixin.py:109} INFO - [2022-01-14 20:54:30,441] {dag.py:2935} INFO - Setting next_dagrun for parallel_dag.processing_tasks to None
[2022-01-14 20:54:31,182] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/parallel_dags.py took 7.335 seconds
